Source/Flow/Private/Nodes/FlowNode.cpp

```diff
@@ -148,6 +150,45 @@ UFlowAsset* UFlowNode::GetFlowAsset() const
 	return GetOuter() ? Cast<UFlowAsset>(GetOuter()) : nullptr;
 }
 
+AActor* UFlowNode::TryGetRootFlowActorOwner() const
```

Comment:
_This code moved from FlowNode_PlayLevelSequence into an UFlowNode function so that other FlowNodes can use it._